<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>book_menu</class>
 <widget class="QWidget" name="book_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>530</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_7">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/b6aca1fc824cc2b1ab0b7d294038e47.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/239b60a65d18d132ec6e1216140e922.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/faa9ca0bbeda6c6359c0196065df5eb.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/63e14385b2456da5cb2a54246a864f7.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/ff17562ded6702cd78d587e81b8b2f8.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/6ac8954a2f2aca6d9688792179c4e69.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_8">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/ec9e7b00725daed03a20ebf21e77e63.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/64ba34539fc4491917630492fd908b7.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>430</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Completed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>mapo tofu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>beef</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>set meal 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>set meal 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>400</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>beef noodles</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>set meal 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>set meal 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>chicken</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>$9.99</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>$19.99</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>$17.99</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>430</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>$15.99</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>$16.99</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>$18.99</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>$20.99</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>$10.99</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>9</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Book Menu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
